magefiles/magefile.go

```diff
@@ -51,7 +51,7 @@ var (
 	// can be periodic, presubmit or postsubmit
 	jobType                    = utils.GetEnv("JOB_TYPE", "")
 	reposToDeleteDefaultRegexp = "jvm-build|e2e-dotnet|build-suite|e2e|pet-clinic-e2e|test-app|e2e-quayio|petclinic|test-app|integ-app|^dockerfile-|new-|^python|my-app|^test-|^multi-component"
-	repositoriesWithWebhooks   = []string{"devfile-sample-hello-world", "hacbs-test-project"}
+	repositoriesWithWebhooks   = []string{"devfile-sample-hello-world", "hacbs-test-project", "secret-lookup-sample-repo-two"}
```

Comment:
I have a gitlab repo where webhooks pile up as well, does this nightly job support clean up of the GitLab repos as well? (after reading the code, doesn't look like it)

Reply:
it would have to be updated for the purpose of using it for gitlab

Reply:
good to know, I've added a note for it in this ticket [STONEINTG-937](https://issues.redhat.com/browse/STONEINTG-937)